xpad.c

```diff
@@ -952,6 +961,36 @@ static void xpadone_process_packet(struct usb_xpad *xpad, u16 cmd, unsigned char
 				 (__u16) le16_to_cpup((__le16 *)(data + 8)));
 	}
 
+	/* paddle handling */
+	/* based on SDL's SDL_hidapi_xboxone.c */
+	if (xpad->mapping & MAP_PADDLES) {
+		/* HACK: Paddle data is delivered differently for different models/firmwares. There's probably a better way to detect this... */
+		if (xpad->name == "Microsoft X-Box One Elite pad"){
+			/* Mute paddles if controller has a custom mapping applied */
+			/* Checked by comparing the current mapping config against the factory mapping config */
+			if (memcmp(&data[4], &data[18], 2) != 0) {
```

Comment:
I took this detection code almost verbatim from SDL's implementation, so I'm _pretty sure_ it works... nonetheless, it is the part of this PR that I'm the most concerned about.

I'd love it if anyone with an OG Elite controller could verify that custom profiles correctly block paddle inputs!